subsys/suit/provisioning/soc/Kconfig.nrf54h20

```diff
@@ -39,7 +39,7 @@ rsource "Kconfig.template.manifest_config"
 endmenu
 
 manifest=APP_RECOVERY
-manifest-class=nRF54H20_sample_app_recovery
+manifest-class=nRF54H20_app_recovery
```

Comment:
Please remove the `default y` for recovery MPIs. Currently the ROOT manifest is not used as recovery manifest in case of missing recovery manifest.

Reply:
Fixed here: https://github.com/nrfconnect/sdk-nrf/pull/15639 - additionally, added https://github.com/ahasztag/sdk-nrf/blob/c13c4ba5c14751933190970008178624dc1422e9/sysbuild/CMakeLists.txt#L405-L408 to achieve MPI generation when building with recovery